src/components/20-molecules/file-upload/index.js

```diff
@@ -251,78 +296,153 @@ class AXAFileUpload extends LitElement {
         : '';
   }
 
-  validateFiles(compressedImages, notImagesFiles) {
-    const { maxSizeOfSingleFileKB, maxNumberOfFiles } = this;
+  validateFiles(compressedImages, notImagesFiles, droppedFilesWithID) {
+    const { maxSizeOfSingleFileKB } = this;
     const maxSizeOfSingleFileInBytes = getBytesFromKilobyte(
       maxSizeOfSingleFileKB
     );
 
-    const faultyFiles = [];
-    const finalFiles = [];
-    const files = [...notImagesFiles].concat(compressedImages);
+    const faultyCompressedFiles = [];
+    const validCompressedFiles = [];
+    const faultyOriginalFiles = [];
+    const validOriginalFiles = [];
```

Comment:
Kind of complex with all these arrays. Hard to read this function before first coffee :)